src/lib/y2network/presenters/interfaces_summary.rb

```diff
@@ -50,6 +53,115 @@ def text
 
         Yast::Summary.DevicesList(overview)
       end
+
+      # Generates a one line text summary for the configured interfaces.
+      #
+      # @example with one configured interface
+      #   presenter.one_line_text
+      #   => "DHCP / eth1"
+      #
+      # @example with many configured interfaces
+      #   presenter.one_line_text
+      #   => "Multiple Interfaces"
+      #
+      # @return [String] summary in just one line and in plain text
+      def one_line_text
+        protocols = []
+        output = []
+
+        protocols << "DHCP" if config.connections.any?(&:dhcp?)
+        protocols << "STATIC" if config.connections.any?(&:static?)
+
+        output << protocols.first if protocols.uniq.size == 1
+
+        case config.connections.size
+        when 0
+          return Yast::Summary.NotConfigured
+        when 1
+          output << config.connections.first.interface
+        else
+          output << _("Multiple Interfaces")
+        end
+
+        output.join(" / ")
+      end
+
+      # Generates a summary in RichText format for the configured interfaces
```

Comment:
NP: I'd say in HTML format

Reply:
Is not HTML. Is our own RichText format, the same that our RichText widget supports.

Reply:
Then... let's use always the same term. But wait, it just a NP :)